File: Backend/API/src/main/java/com/example/localhistory/landmark/dto/response/LandmarkDTO.java
package com.example.localhistory.landmark.dto.response;

import com.example.localhistory.coordinates.Coordinates;
import lombok.Data;

import java.time.LocalDateTime;

@Data
public class LandmarkDTO {
    private Long id;
    private String title;
    private String description;
    private LocalDateTime dateCreated;
    private String imageUrl;
    private Coordinates coordinates;
    private Long ownerId;
    private Integer visitRewardPoints;
    private Integer visitsCount;
    private Long quizId;
}

File: Frontend/app/src/main/java/com/example/localhistory/model/response/LandmarkDTO.kt
package com.example.localhistory.model.response

import com.example.localhistory.model.Coordinates
import kotlinx.datetime.LocalDateTime

data class LandmarkDTO(
    val id: Long,
    val title: String,
    val description: String,
    val dateCreated: LocalDateTime?,
    val imageUrl: String,
    val coordinates: Coordinates,
    val ownerId: Long?,
    val visitRewardPoints: Int,
    val visitsCount: Int
)
